ctapipe/core/traits.py

```diff
@@ -124,3 +128,135 @@ def has_traits(cls, ignore=("config", "parent")):
     here.
     """
     return bool(set(cls.class_trait_names()) - set(ignore))
+
+
+class TelescopeParameter(List):
+    """
+    Allow a parameter value to be specified as a simple float, or as a list of
+    patterns that match different telescopes.
+    The patterns are given as a list of 3-tuples in in the
+    form: `[(command, argument, value), ...]`.
+
+    Command can be one of:
+    - '*' match all telescopes,  Here argument is ignored.
```

Comment:
Why give it then? Isn't it possible two just use a 2 tuple for `*`?

Reply:
I decided to make it symmetric, so all commands are 2-tuples (makes it also easier to convert to a table, etc, later).

Reply:
I could get rid of the "*" command altogether, and just replace it with either:
 `("type", "*", 2.5)`
or
`("id", "*", 2.5)`

Would that be better? So far only "*" or a type or id would be allowed, but we could later implement true globs, like `("type","LST*", 2.5)`